src/components/codeblock.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { Box } from '@blockstack/ui';
+import css from '@styled-system/css';
+import { Highlighter } from '@components/highlighter';
+
+export const CodeBlock = ({ code }: { code: string }) => {
+  const [editorCode] = useState(code?.toString().trim());
+
+  return (
+    <Box
+      // @ts-ignore
```

Comment:
What's the error without this?

Reply:
css isn't currently a prop that be passed to any element for some reason. It's built into styled-components, and I haven't determined how to type it yet. You should be able to pass `css={css({..})}` and it will turn into a styled-component via the babel plugin.

Reply:
More and more the sx={} seems like a good idea

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/31245#issuecomment-586480973

Reply:
Yeah, I want to add support for the sx prop, but it's not a straight port over to theme-ui. Involves some more complex adjustments. I'll make an issue to explore it though in blockstack/ui!